### PR TITLE
Adding To Cart methods/Test

### PR DESCRIPTION
**Pull request by SerineBetraoui**
Added selectProduct method in ShoppingPage, accepts two int parameters for product ID and quantity

Adjusted addToCartUserSignedIn method, asserts 200 code and prints response body FOREVER

### CODE REVIEW

**Review by jacktegart — APPROVED** [review]

**Review by SuzyLady — APPROVED** [review]

File: src/test/java/PageObjects/ShoppingPage.java [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+        formData.put("id_product", productId);
+        formData.put("id_customization", "0");
+        formData.put("group[1]", "1");
+        formData.put("group[3]", "13");
+        formData.put("qty", productQuantity);
+        formData.put("add", "1");
+        formData.put("action", "update");
+
+
+        response = httpRequest.filter(cookieFilter)
```
Comment (Leosaw) [inline src/test/java/PageObjects/ShoppingPage.java:20]: we will use the sendpost method yoshi made

**Review by Aayush-Srikanth — APPROVED** [review]
Nice one!

File: src/test/java/Tests/AddToCartTest.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    SignInPage signInPage = new SignInPage();
+    ShoppingPage shoppingPage = new ShoppingPage();
+
+    @Test
+    public void addToCartUserSignedIn() {
+        homePage.getHomePage();
+        signInPage.signInValidManualAccDetails();
+        shoppingPage.selectProduct(6, 1);
+        homePage.assertStatusCode(200);
+        System.out.println(homePage.getResponseBody());
```
Comment (Leosaw) [inline src/test/java/Tests/AddToCartTest.java:19]: need to add an assertion base on the cart page for robustness.

File: src/test/java/PageObjects/ShoppingPage.java (hunk 1 above)
Comment (Aayush-Srikanth) [inline src/test/java/PageObjects/ShoppingPage.java:20]: - May be worth using the sendPostRequest (in BasePage) method to clean up the code a bit (can use a HashMap for the queryParams)

**Review by Leosaw — APPROVED** [review]
Awesome job @SerineBetraoui @SuzyLady @jacktegart

**Pull request merged**